x/axelarnet/keeper/msg_server.go

```diff
@@ -326,11 +326,6 @@ func (s msgServer) AddCosmosBasedChain(c context.Context, req *types.AddCosmosBa
 		if err := s.nexus.RegisterAsset(ctx, chain, asset, utils.MaxUint, types.DefaultRateLimitWindow); err != nil {
 			return nil, err
 		}
-
```

Comment:
What's the reason for these removals in this file?

Reply:
bad practice to ignore the error (which is needed here since it could be triggered multiple times). And it allows less flexibility over how to register the asset on axelarnet.